dali/CMakeLists.txt

```diff
@@ -52,6 +54,7 @@ if(${CMAKE_VERSION} VERSION_LESS "3.9.0")
   set(CUDA_LIBRARIES ${CUDA_LINK_LIBRARIES_KEYWORD} ${CUDA_LIBRARIES})
 endif()
 cuda_add_library(${dali_operator_lib} STATIC ${DALI_OPERATOR_SRCS})
+target_link_libraries(${dali_operator_lib} PRIVATE "${dali_kernel_lib}")
```

Comment:
It is only to provide headers?
Because `dali_operator_lib` is static so it doesn't to link with anything.